src/Iuliia/Obstacles

```diff
@@ -0,0 +1 @@
+Obstacles are what you see when you take your eye off your goal.
```

Comment:
If you can find ways without any obstacles, it probably leads nowhere. - Che Guevara